draft-ietf-jsonpath-base.md

```diff
@@ -266,11 +266,11 @@ Selector:
 
 ### JSON Values as Trees of Nodes
 
-This specification models the argument as a tree of JSON values, each
+This document models the argument as a tree of JSON values, each
```

Comment:
Another way to write this is "We model...".

Reply:
True. I wanted to avoid that voice because the tone can sometimes come across as a little superior.